examples/slate-demo.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates basic usage of the firefly_client API for Firefly Slate Viewer.\n",
+    "\n",
+    "Note that it may be necessary to wait for some cells (like those displaying an image) to complete before executing later cells."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for Python 2/3 compatibility"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function, division, absolute_import"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for firefly_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from firefly_client import FireflyClient"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import firefly_slate_demo as fs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example tentatively uses localhost:8080 as the server\n",
+    "\n",
+    "'slate.html' is a template made for grid view "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "host='localhost:8080'\n",
+    "channel = 'myChannel8'\n",
+    "html = 'slate.html'\n",
+    "\n",
+    "fc = FireflyClient(host, channel=channel, html_file=html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dispaly tables, images, XY charts, and Histograms in Window/Grid like layout\n",
+    "\n",
+    "Each rendering unit on Firefly Slate Viewer is called a'cell'. To render a cell and its content, the cell location (row, column, width, height) and a cell ID are needed.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use astropy here for convenience, but firefly_client itself does not depend on astropy."
```

Comment:
`astropy.utils.data` isn't used in this notebook -- so I think it would be better to insert this in `firefly_slate_demo.py` as a comment and to remove this cell and the import of astropy.utils.data in the notebook.